.eslintrc.json

```diff
@@ -1,235 +1,219 @@
 {
-    "root": true,
-    "extends": [
-      "react-app",
-      "airbnb",
-      "plugin:jsx-a11y/recommended",
-      "plugin:import/errors",
-      "plugin:import/warnings",
-      "plugin:import/typescript",
-      "plugin:prettier/recommended",
-      "plugin:testing-library/recommended",
-      "plugin:testing-library/react",
-      "plugin:jest-dom/recommended",
-      "eslint:recommended",
-      "plugin:@typescript-eslint/recommended"
+  "root": true,
+  "extends": [
+    "react-app",
+    "airbnb",
+    "plugin:jsx-a11y/recommended",
+    "plugin:import/errors",
+    "plugin:import/warnings",
+    "plugin:import/typescript",
+    "plugin:prettier/recommended",
+    "plugin:testing-library/recommended",
+    "plugin:testing-library/react",
+    "plugin:jest-dom/recommended",
+    "eslint:recommended",
+    "plugin:@typescript-eslint/recommended"
+  ],
+  "overrides": [
+    {
+      "files": ["*.tsx"],
+      "extends": [
+        "plugin:@typescript-eslint/recommended-requiring-type-checking"
+      ]
+    }
+  ],
+  "plugins": [
+    "jsx-a11y",
+    "prettier",
+    "testing-library",
+    "jest-dom",
+    "@typescript-eslint"
+  ],
+  "parser": "@typescript-eslint/parser",
+  "parserOptions": {
+    "project": "tsconfig.json"
+  },
+  "rules": {
+    "no-use-before-define": "off",
+    "prettier/prettier": [
+      "error",
+      {
+        "semi": true,
+        "trailingComma": "all",
+        "singleQuote": false,
+        "printWidth": 80
+      }
```

Comment:
This file was auto formatted... this is the pertinent change.

Reply:
noice - one config and script to rule them all :)